From 4592eac08fb8f1f9682a4f8e8cb00eb3a7bdfbcd Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Wed, 12 Feb 2014 10:18:30 +0000
Subject: [PATCH] Install thermald in ${sbindir}, and point systemd service to
 it

Substituting @sbindir@ in thermald.service.in means that the systemd
unit will work if compiled with a non-default --sbindir, or a
non-/usr prefix.

Distributions that don't distinguish between /usr/sbin and /usr/bin
can configure with --sbindir=/usr/bin or similar.
---
 Makefile.am                                  | 2 +-
 data/thermald.service.in                     | 2 +-
 distribution_integration/thermal-daemon.spec | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 981972c..57a2057 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -23,7 +23,7 @@ EXTRA_DIST=Makefile.glib \
 	thermald.pc.in
 
 # Programs to build
-bin_PROGRAMS = thermald
+sbin_PROGRAMS = thermald
 
 # Evaluate Table Application
 thermald_CPPFLAGS = \
diff --git a/data/thermald.service.in b/data/thermald.service.in
index 6d01518..1d5a704 100644
--- a/data/thermald.service.in
+++ b/data/thermald.service.in
@@ -5,7 +5,7 @@ After=syslog.target
 [Service]
 Type=dbus
 BusName=org.freedesktop.thermald
-ExecStart=/usr/bin/thermald --no-daemon --dbus-enable
+ExecStart=@sbindir@/thermald --no-daemon --dbus-enable
 StandardError=null
 
 [Install]
diff --git a/distribution_integration/thermal-daemon.spec b/distribution_integration/thermal-daemon.spec
index d2b9f38..c9f905c 100644
--- a/distribution_integration/thermal-daemon.spec
+++ b/distribution_integration/thermal-daemon.spec
@@ -47,7 +47,7 @@ make %{?_smp_mflags}
 %systemd_postun_with_restart thermald.service
 
 %files
-%{_bindir}/thermald
+%{_sbindir}/thermald
 %config(noreplace) %{_sysconfdir}/dbus-1/system.d/org.freedesktop.thermald.conf
 %{_datadir}/dbus-1/system-services/org.freedesktop.thermald.service
 %config(noreplace) %{_sysconfdir}/thermald/thermal-conf.xml
